### PR TITLE
[TEST] Fix unstable LZ4 unit test

### PR DESCRIPTION
**Pull request by FMX**
# [BUG]Fix unstable unit test.

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

**Comment by FMX** [comment]
\regression

**Comment by pan3793** [comment]
Have you identified the root cause of the failure?

File: worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ClusterReadWriteTestWithLZ4.scala [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      "celeborn.master.endpoints" -> s"localhost:$masterPort")
+    setUpMiniCluster(masterConf, workerConf)
+  }
+
+  override def afterAll(): Unit = {
+    println("test done")
+    sys.exit(0)
+  }
+
+  test(s"test MiniCluster With ZSTD") {
```
Comment (waitinfuture) [inline worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ClusterReadWriteTestWithLZ4.scala:51]: lz4？

File: worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ReadWriteTestBase.scala [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import org.apache.celeborn.client.compress.Compressor.CompressionCodec
+import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.identity.UserIdentifier
+import org.apache.celeborn.common.internal.Logging
+
+trait ReadWriteTestBase extends Logging {
+  val masterPort = 19097
+
+  def testReadWriteByCode(compressionCodec: CompressionCodec): Unit = {
+    val codec = CompressionCodec.LZ4
```
Comment (waitinfuture) [inline worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ReadWriteTestBase.scala:37]: fixed to lz4?

**Review by waitinfuture — APPROVED** [review]
LGTM.

File: worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ClusterReadWriteTestWithZSTD.scala [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    val masterConf = Map(
+      "celeborn.master.host" -> "localhost",
+      "celeborn.master.port" -> masterPort.toString)
+    val workerConf = Map(
+      "celeborn.master.endpoints" -> s"localhost:$masterPort")
+    setUpMiniCluster(masterConf, workerConf)
+  }
+
+  override def afterAll(): Unit = {
+    println("test done")
```
Comment (pan3793) [inline worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ClusterReadWriteTestWithZSTD.scala:47]: please avoid using println

**Pull request merged**